setup.cfg

```diff
@@ -65,6 +65,7 @@ exclude =
     mitmproxy/net/http/multipart.py
     mitmproxy/net/tcp.py
     mitmproxy/net/tls.py
+    mitmproxy/net/udp.py
```

Comment:
The main intention of this exclude list was to be temporary - so unless there really is no way around it of having 100% individual test coverage, I would like to see this list here shrink over time (and not grow).

Reply:
The transparent proxying bits of `net/udp.py` are not really testable without lots of monkeypatching, which is a bit pointless. I guess we can exclude those liberally with `# pragma: no cover` statements and remove the entry here.